app/frontend/src/features/messages/groupchats/groupChatsActions.ts

```diff
@@ -0,0 +1,74 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { User } from "../../../pb/api_pb";
+import { GroupChat } from "../../../pb/conversations_pb";
+import { RootState } from "../../../reducers";
+import { service } from "../../../service";
+import { groupChatsFetched, messagesFetched, groupChatSet } from ".";
+
+type FetchGroupChatsArguments = void;
+
+export const fetchGroupChats = createAsyncThunk<
+  void,
+  FetchGroupChatsArguments,
+  { state: RootState }
+>("hostRequests/fetchGroupChats", async (_, thunkApi) => {
+  const groupChats = await service.conversations.listGroupChats();
+  thunkApi.dispatch(groupChatsFetched(groupChats));
+});
+
+export const fetchMessages = createAsyncThunk<
+  void,
+  GroupChat.AsObject,
+  { state: RootState }
+>("hostRequests/fetchMessages", async (groupChat, thunkApi) => {
+  const messages = await service.conversations.getGroupChatMessages(
+    groupChat.groupChatId
+  );
+  thunkApi.dispatch(messagesFetched(messages));
+});
```

Comment:
```suggestion
export const fetchMessages = createAsyncThunk("hostRequests/fetchMessages", async (groupChat) => {
  return service.conversations.getGroupChatMessages(
    groupChat.groupChatId
  );
});
```
Same as first reason as above.